ndk/src/event.rs

```diff
@@ -335,72 +321,59 @@ pub enum ToolType {
     __Unknown(i32),
 }
 
-/// A bitfield representing the state of buttons during a motion event.
-#[derive(Copy, Clone, Debug, PartialEq, Eq)]
-pub struct ButtonState(pub u32);
-
-impl ButtonState {
-    #[inline]
-    pub fn primary(self) -> bool {
-        self.0 & ffi::AMOTION_EVENT_BUTTON_PRIMARY != 0
-    }
-    #[inline]
-    pub fn secondary(self) -> bool {
-        self.0 & ffi::AMOTION_EVENT_BUTTON_SECONDARY != 0
-    }
-    #[inline]
-    pub fn teriary(self) -> bool {
-        self.0 & ffi::AMOTION_EVENT_BUTTON_TERTIARY != 0
-    }
```

Comment:
Note that this PR will fix the `teriary` -> `TERTIARY` typo as pointed out in https://github.com/rust-mobile/android-activity/pull/163, by freshly copying the constants from the header file.